mobify/platform-scaffold
# Build and test on Windows CI using AppVeyor

A first pass at using [AppVeyor](https://ci.appveyor.com/projects) to install, build, and test on a Windows environment.

## Changes
- Add `appveyor.yml` config

## How to test-drive this PR
- Examine AppVeyor build output and ensure that there are no errors, eg https://ci.appveyor.com/project/ellenmobify/platform-scaffold/build/1.0.63
- Verify CircleCI Builds are not affected
